ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/TokenAuthentication/TokenAuthentication.h

```diff
@@ -38,11 +40,14 @@ class TokenAuthentication : public QQuickItem
 
   void componentComplete() override;
   static void init();
-  Q_INVOKABLE void loadSecuredLayer();
 
 private:
+  QUrl m_portalURL = QUrl("https://www.arcgis.com/");
+  QString m_itemID = "e5039444ef3c48b8a8fdc9227f9be7c1";
   Esri::ArcGISRuntime::Map* m_map = nullptr;
   Esri::ArcGISRuntime::MapQuickView* m_mapView = nullptr;
+  Esri::ArcGISRuntime::Portal* m_portal = nullptr;
+  Esri::ArcGISRuntime::PortalItem* m_portalItem = nullptr;
```

Comment:
These shouldn't need to be included in the header file. Take a look at how I did it with the Geotriggers sample: https://github.com/Esri/arcgis-runtime-samples-qt/blob/v.next/ArcGISRuntimeSDKQt_CppSamples/Analysis/Geotriggers/Geotriggers.cpp#L51

Reply:
@tanneryould is your concern having these as class members, or initializing them to `nullptr`?

Reply:
@JamesMBallard having these as class members - is there a point to declaring all of these members instead of just writing the following?
```cpp
m_map = new Map(new Portal("e5039444ef3c48b8a8fdc9227f9be7c1", this), this);
```

Reply:
@tanneryould good point. That makes sense to me to just utilize them where needed, once. Thanks for clarifying.

Reply:
I think avoiding having these as members is a good suggestion but I would still recommend naming the variables like the portal item Id and also the url for arcgis.com. We want to make sure the code is easy to read and straightforward for users to swap in there own portals/items.

Reply:
Thanks for your comments and suggestions @tanneryould, @JamesMBallard, and @lsmallwood. I have kept the portalURL, itemID, m_portal, and m_portalItem variables. Although this is not the most efficient implementation, I think explicitly declaring each of the these as variables helps demonstrate how the authentication is set-up and makes the code readable and re-usable.

Reply:
@tanneryould and @JamesMBallard, following my comment above and merging this PR, I discussed your suggestions with @lsmallwood and realised I had not fully understood them. To remedy this, I have made some further changes to remove m_portal and m_portalItem from the member variables. These changes can be found in https://github.com/Esri/arcgis-runtime-samples-qt/pull/1303

The `Portal` and `PortalItem` objects are no longer member variables, and are created as local variables in the constructor. These local variables are then used to create the `Map` object. Hopefully the changes in the above PR are consistent with your suggestions.

Thanks again!